linode/manager
# refactor: [M3-7547] - Graphs stories v7 migration

## Description 📝
PR to migrate our Graph stories to v7

## Changes  🔄
- Migrate GaugePercent and add test
- Migrate LineGraph and add test
- Update storybook to `7.6.4`
- Add `@storybook/addon-storysource` to show source code to individual stories

## Preview 📷
![Screenshot 2023-12-14 at 16 08 06](https://github.com/linode/manager/assets/130582365/bd97206c-10ed-4cbf-b43a-3a3694d7ee07)

![Screenshot 2023-12-14 at 16 08 39](https://github.com/linode/manager/assets/130582365/572f9dfc-b792-4059-a4e5-90365a27c877)

## How to test 🧪

### Verification steps 
(How to verify changes)
- run `yarn storybook`
- verify http://localhost:6006/?path=/docs/components-graphs-gaugespercent--documentation
- verify http://localhost:6006/?path=/docs/components-graphs-linegraph--documentation
- run `yarn test GaugePercent.test.tsx`
- run `yarn test /LineGraph.test.tsx`

## As an Author I have considered 🤔

*Check all that apply*
